### DISCUSSION TITLE
Github actions package publish error

### DISCUSSION THREAD

**Discussion by KInGATiLLa** [npm/feedback#365] [Registry] [answered]
I'm using github actions by publishing packages. When i run action it gives me this error. But why npm do publish action 2 times?
Is it runs must be only once?

![image](https://user-images.githubusercontent.com/17881842/116819779-22b1dc80-aba4-11eb-9470-0691f55164cf.png)

And later on I go to my npmjs profile page it published new version. Thank you for your help!

**Comment by KInGATiLLa** [comment]
Full log here
![image](https://user-images.githubusercontent.com/17881842/116820020-7670f580-aba5-11eb-81c8-e3a2174348a2.png)

**Comment by ljharb** [comment]
My guess is that you're running the action on both `push` and `pull_request`, or possibly on more branches than just the default branch.

**Comment by subnwa** [comment]
Your account must have email verification and 2-factor verification turned on

**Reply by subnwa** [reply]
or you need to change the version in that `package.json` file because it changes version every time you use the `publish` command

**Comment by MylesBorins** [comment, marked as answer]
Hey @KInGATiLLa. This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. 

as mentioned by others in the thread it would appear that you are trying to publish a version already published to the registry as you can see as the 403 error
